tests/test_end_to_end.py

```diff
@@ -1,10 +1,11 @@
 import importlib.resources as resources
-from unittest import mock
+import os
+from pathlib import Path
 
 import numpy as np
```

Comment:
numpy was removed as a dependency. is it pulled in through another dependency?

Reply:
if it's imported, it should be an explicit dependency. I've added it back in.